client/store/modules/auth/getters.js

```diff
@@ -2,3 +2,5 @@ import get from 'lodash/get';
 import { role } from 'shared';
 
 export const isAdmin = ({ user }) => get(user, 'role') === role.user.ADMIN;
+
+export const token = state => state.token;
```

Comment:
Whats the point of this getter ?

Reply:
I added that getter because I had to get the `token` and pass it as the argument to `SSEClient` constructor. Feel free to take a look [here](https://github.com/ExtensionEngine/tailor/pull/367/files#diff-24ace95a81a97a92f1e785a22e295893R16).
 
Another reason I needed that getter is because I had to add the same token to the Axios interceptor for the authorization.
References:
 - [Reference_1](https://github.com/ExtensionEngine/tailor/pull/367/files#diff-096103d8490b885f9d5037baea6aea80R58)
- [Reference_2](https://github.com/ExtensionEngine/tailor/pull/367/files#diff-de413e6307dae35c58e236129ed45f76R19)

Reply:
Sorry for not clarifying it a bit better 😞, I was referring to the fact that we exposed it as a getter although we are not doing any processing. Can we just pick it up from state?